packages/adblocker/src/engine/engine.ts

```diff
@@ -674,6 +765,8 @@ export default class FilterEngine extends EventEmitter<
     let allowGenericHides = true;
     let allowSpecificHides = true;
 
+    const isFilterExcluded = (filter: IFilter) => this.preprocessors.isFilterExcluded(filter);
```

Comment:
why this is defined multiple times? couldn't it be a property on the engine? (need to be bound if we want to pass it around)

```class Engine {
  isFilterExcluded = (filter: IFilter) =>this.preprocessors.isFilterExcluded(filter)
}
```

Reply:
The code has been changed to use internal variable. Originally, I intended to set this as a `private` variable but set to `public` to resolve the problem on lazy-initialization on deserialization step of engine.